met-web/src/apiManager/httpRequestHandler/index.ts

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import UserService from '../../services/userService';
+
```

Comment:
You could have a more typescript approach which is having a exporting a default class with multiple methods

Reply:
done